### DISCUSSION TITLE
Install without ESPHome?

### DISCUSSION THREAD

**Discussion by surfingreg** [echavet/MitsubishiCN105ESPHome#203] [General]
Apologies if this is a dumb question. But as I don't have a "supervised" version of Home Assistant, am only on mac and USB can't pass throught to Docker, I'm finding the install of ESPHome to be...prohibitive. 'brew install esphome' has been running for an hour plus. Anyway, is it not possible to use all the work that's been done here in the Arduino IDE? Like the original SwiCago repo (but for ESP32)? Thanks again, this is awesome work.

**Comment by echavet** [comment]
try 

`pip install --upgrade esphome`

on your mac

then 
```
esphome compile yourfile.yaml
esphome upload yourfile.yaml
```

**Reply by surfingreg** [reply]
Thanks. That got me going a bit.
